inference-engine/tests/ngraph_functions/src/low_precision_transformations/clamp_function.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph_functions/low_precision_transformations/clamp_function.hpp"
+
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph_ops/type_relaxed.hpp>
+#include "ngraph_functions/subgraph_builders.hpp"
+#include "transformations/low_precision/network_helper.hpp"
+
+namespace ngraph {
+namespace builder {
+namespace subgraph {
+    std::shared_ptr<ngraph::Function> ClampFunction::getOriginal(
+        const ngraph::element::Type originalFunctionPrecision,
+        const ngraph::Shape& inputShape,
+        const ActualValues& values) {
+        const auto input = std::make_shared<ngraph::opset1::Parameter>(values.lowPrecision, ngraph::Shape(inputShape));
+        std::shared_ptr<ngraph::Node> parent = input;
+
+        const std::shared_ptr<ngraph::Node> convert = std::make_shared<ngraph::opset1::Convert>(parent, originalFunctionPrecision);
+        parent = convert;
+
+        if (!values.subtractValues.empty()) {
+            auto constant = std::make_shared<ngraph::opset1::Constant>(originalFunctionPrecision,
+                Shape({ values.subtractValues.size() }), values.subtractValues);
+            const std::shared_ptr<ngraph::Node> subtract = std::make_shared<ngraph::opset1::Subtract>(
+                parent,
+                constant);
+            parent = subtract;
+        }
+
+        const std::shared_ptr<ngraph::Node> multiply = std::make_shared<ngraph::opset1::Multiply>(
+            parent,
+            std::make_shared<ngraph::opset1::Constant>(originalFunctionPrecision, Shape({ values.multiplyValues.size() }), values.multiplyValues));
+        parent = multiply;
+
+        const std::shared_ptr<ngraph::Node> clamp = std::make_shared<ngraph::opset1::Clamp>(parent, 0.0, 10.0);
+
+        ngraph::ResultVector results{ std::make_shared<ngraph::opset1::Result>(clamp) };
+        return std::make_shared<ngraph::Function>(results, ngraph::ParameterVector{ input }, "ClampTransformation");
+    }
+
+    std::shared_ptr<ngraph::Function> ClampFunction::getReference(
+        const ngraph::element::Type originalFunctionPrecision,
+        const ngraph::Shape& inputShape,
+        const ExpectedValues& values) {
+        auto input = std::make_shared<ngraph::opset1::Parameter>(originalFunctionPrecision, ngraph::Shape(inputShape));
```

Comment:
use `values.lowPrecision` instead `originalFunctionPrecision`. Cases:
1. low precision is expected if LayerTransformation::Param::updatePrecisions = true
2. original function precision is expected if LayerTransformation::Param::updatePrecisions = false

Note, please, don't pass LayerTransformation::Param instance into test

Reply:
Done, thanks!